src/app/pages/signup/signup.component.spec.ts

```diff
@@ -11,7 +11,7 @@ describe('SignupComponent', () => {
       imports: [SignupComponent]
     })
     .compileComponents();
-    
+    console.log('testing signup component');
```

Comment:
Please use alert instead of this comment

Reply:
i will make this change.